rai/lib/utility.cpp

```diff
@@ -9,5 +9,5 @@ void release_assert_internal (bool check, const char * check_expr, const char *
 	}
 
 	std::cerr << "Assertion (" << check_expr << ") failed " << file << ":" << line << std::endl;
-	abort();
+	abort ();
```

Comment:
This is already changed in master, I think

Reply:
I rebased but this still shows up after running ci/clang-format-all.sh. Was master force-updated maybe? I don't think there'll be a merge conflict.